### PR TITLE
chore: remove repetitive words

### PR DESCRIPTION
**Pull request by longhutianjie**

### CODE REVIEW

**Comment by nahuhh** [comment]
Is this you? https://github.com/monero-project/monero-site/pull/2338

**Comment by plowsof** [comment]
most likely a bot account farming for merges in popular repos searching for duplicated words. the edit to the chat logs exposes them.

File: _posts/2020-05-06-mrl-meeting.md
Comment (HardenedSteel) [inline _posts/2020-05-06-mrl-meeting.md:1]: I think IRC logs shouldn't be edited

**Comment by nahuhh** [comment]
@plowsof the other pr was actually updated and did the same thing. So either ai is getting smarter (responding and removing some stuff), or there is someone monitoring

**Pull request closed without merging**